source/draw/RasterMonochrome.ooc

```diff
@@ -33,6 +33,7 @@ RasterMonochrome: class extends RasterPacked {
 	create: func (size: IntSize2D) -> Image { This new(size) }
 	copy: func -> This {
 		result := This new(this)
+		result _buffer = ByteBuffer new(this _buffer size)
```

Comment:
Sorry, I was refering to the constructor in `RasterPacked`:
```ooc
init: func ~fromOriginal (original: RasterImage)
```

We cannot do anything with the buffer in `original` here unless it explicitly is set to be of type `RasterPacked`. But for it to be, we also have to update the appropriate constructor in each subclass since they expect the same type:
```ooc
RasterMonochrome: class extends RasterPacked {
// ...
init: func ~fromRasterImage (original: RasterImage) { super(original) }
// ...
```
Or, add a new `init ~fromRasterPacked`.